app/players/player.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    def __int__(self, nickname: str) -> None:
```

Comment:
There is a typo in the constructor method name. It should be `__init__` instead of `__int__`.